Rename `data` to `datasets` to avoid confusion (#29)
alphagov/datagovuk-tech-docs: source/documentation/publish_and_manage_data/add_data.md

# Add datasets manually

> If you don’t want to [automatically harvest datasets](../harvest_data), you can add a dataset to data.gov.uk using an online form. You will need to submit one form per dataset. Please note that whilst datasets can be created manually, there is no manual process for adding data to them; this must be done through data harvesting.

Reviewer: > Please note that whilst datasets can be created manually, there is no manual process for adding data to them; this must be done through data harvesting.

This is incorrect.

The datasets contain one or more links to external URLs (called 'resources' in CKAN), where the data can be downloaded from.  If the dataset is added manually, the publisher enters the URL by hand.  If the dataset is harvested, we populate the URL automatically from their harvest source.  Either way, the publisher is responsible for hosting their data files somewhere and then giving us the URL for that data (either manually or through harvesting).

The exception is organograms, but that is covered elsewhere in the docs.

Author: That's what I'm saying here, but sounds like it needs to be reworded to avoid confusion. How about this?

```suggestion
Please note that whilst datasets can be created manually, there is no manual process for uploading data to them; this can only be done through data harvesting.
```

Reviewer: That still doesn't read correctly to me.  Whether harvested or manually entered, there's no way of uploading data to DGU (except organograms).

The publisher has to upload the data on their own website (e.g. a local council site, or on GOV.UK).  It may be as simple as uploading a CSV file, then pasting the URL into DGU's CKAN.

Author: Hmm, how about:

> Please note that whilst datasets can be created manually, there is no way of manually uploading data into the dataset. You have to host the data elsewhere and add that URL as a 'Data Link' resource so that it can be made visible on data.gov.uk.

Reviewer: That would be correct.  I'm not sure if we actually need to state this here though (it is already explained at https://guidance.data.gov.uk/publish_and_manage_data/).  Have we had any user queries where they are asking how to upload data?

Author: We have had a few queries along the lines of "how do I upload data", "where is the upload button" etc.

Reviewer: This still doesn't feel like the right place to be adding this text, since both harvesting publishers and manual publishers have to host their data externally.  This paragraph is specifically telling users about an alternative to harvesting, but the way they host their data is the same in both cases.

Maybe we should better explain what fields should be filled in, and with what information, on the line `2. Complete the fields.`?

Author: Hmm I think my biggest issue with the current page is the heading, "Add data manually", which I think should be "Add datasets manually" to avoid confusion. Are you happy for me to reduce the scope of my change to just the heading?

Reviewer: Sure.  I think data and datasets are used interchangeably, but it makes sense to be consistent.
